modules/dr-restoring-cluster-state.adoc

```diff
@@ -186,6 +186,186 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
 +
 If the status is `Pending`, or the output lists more than one running etcd pod, wait a few minutes and check again.
 
+. If you have lost the majority of your control plane hosts, leading to etcd quorum loss, delete and recreate the control plane machines (also known as the master machines), one by one. After these machines are recreated, a new revision is forced and etcd scales up automatically.
++
+If you are running installer-provisioned infrastructure, or you used the Machine API to create your machines, follow these steps. Otherwise, you must create the new master using the same method that was used to originally create it.
+
+.. Obtain the machine for the unhealthy member.
```

Comment:
I think in this procedure, it wouldn't necessarily be called the "unhealthy member".  I don't have a good suggestion to make right now, but basically it should be any other control plane node that is not the recovery one.

(Same for anywhere else in this PR that uses "unhealthy"